src/components/table/DataGridModels.ts

```diff
@@ -31,4 +31,5 @@ export interface DataGridState {
   filtering: FilteringRule[];
   sorting: Sorting[];
   search: string;
+  filters: TFilters;
```

Comment:
maybe rename this to availableFilters?
having `filtering` and `filters` props may be confusing when reading code